assembl/tests/model_tests/test_notifications.py

```diff
@@ -98,6 +98,64 @@ def test_notification_follow_synthesis(test_session, discussion,
     assert notification_count == initial_notification_count + 1, "The synthesis post should have matched and created a notification"
 
 
+def test_synthesis_notification_in_user_preferred_language_fr(test_session, discussion,
+                                                           participant1_user, participant1_user_language_preference_fr_cookie,
+                                                           test_app, root_post_1,
+                                                           synthesis_post_1):
+    test_session.flush()
+    subscription = NotificationSubscriptionFollowSyntheses(
+        discussion=discussion,
+        user=participant1_user,
+        creation_origin=NotificationCreationOrigin.USER_REQUESTED
+    )
+    test_session.add(subscription)
+
+    initial_notification_count = test_session.query(Notification).count()
+    dispatcher = ModelEventWatcherNotificationSubscriptionDispatcher()
+
+    notification_count = test_session.query(Notification).count()
+    
+    dispatcher.processPostCreated(synthesis_post_1.id)
+
+    last_notification = participant1_user.notifications[-1]
+
+    html_content = last_notification.render_to_email_html_part()
+
+    assert "subject FR" in html_content
+    assert "introduction FR" in html_content
+    assert "conclusion FR" in html_content
+    # TODO: also check that syntheses of several ideas do also show in the user's preferred language, and other gettext strings if available
+
+
+def test_synthesis_notification_in_user_preferred_language_en(test_session, discussion,
+                                                           participant1_user, participant1_user_language_preference_en_cookie,
+                                                           test_app, root_post_1,
+                                                           synthesis_post_1):
+    test_session.flush()
+    subscription = NotificationSubscriptionFollowSyntheses(
+        discussion=discussion,
+        user=participant1_user,
+        creation_origin=NotificationCreationOrigin.USER_REQUESTED
+    )
+    test_session.add(subscription)
+
+    initial_notification_count = test_session.query(Notification).count()
+    dispatcher = ModelEventWatcherNotificationSubscriptionDispatcher()
+
+    notification_count = test_session.query(Notification).count()
+    
+    dispatcher.processPostCreated(synthesis_post_1.id)
```

Comment:
I am not clear how this doesn't throw an error in travis env, as a lot of methods downstream of this are supposed to be multi-processing. Perhaps they operate on the same thread, therefore they are perfectly okay on the same thread of execution as this test? Not clear to me.

Reply:
Sorry I don't understand, could you explain why it would throw an error? (This method is already used in other tests of this file)

Reply:
Nah, don't worry about it. I just wasn't certain how these methods that are called downstream, which have blocking calls for resources from our database are able to function properly. In any case, this is fine.